### ISSUE TITLE
New start() method in NearlineStorage v7.2

### ISSUE THREAD

**Issue by yhshin** [neicnordic/dcache-endit-provider#7]
Just for heads up, we tested an adapted version of endit-provider at TRIUMF with dCache v7.2.  With 7.2, WatchingEnditNearlineStorage raised exceptions caused by the following one

  java.lang.AbstractMethodError: Receiver class org.ndgf.endit.WatchingEnditNearlineStorage does not define or inherit an implementation of the resolved method 'void start()' of interface org.dcache.pool.nearline.spi.NearlineStorage.

It turned out that NearlineStorage in v7.2 introduced a new start() method

  default void start() throws IOException {}

which was hidden by the private start() method in WENS.  We changed access modifier of start() in WENS to public as a quick fix.  So I'd like to inform that it may need the same or cleaner changes to this too.

**Comment by krishna-veni** [comment]
@yhshin :- you mean we have to change private synchronized void start() method to either default or public in https://github.com/neicnordic/dcache-endit-provider/blob/master/src/main/java/org/ndgf/endit/WatchingEnditNearlineStorage.java ?

**Comment by yhshin** [comment]
Yes it seems to be.  The 'private' access modifier clearly hides new
'default' so I just changed it to 'public' as a quick-and-dirty patch.  BTW
all the other methods of NearlineStorage (NS) interfaced are implemented as
'public' in ListeningNeaelineStorage so it seems to be a right approach to
me.

My guess is that when Gerd wrote the code, there was no 'start' method in
NS interface so he had to create one for Endit-Provider with 'private'
access and to be called for each request to make sure the WatchService is running if
not done yet.  Now it seems that NS calls it so we may not need to check
the service for every request.  Though I haven't checked dCache code thoroughly for
it, I did simply change the access modifier only.
